src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write code here
+const list = document.querySelectorAll('li');
+const number = (str) => {
+  return Number(str.slice(1).split(',').join(''));
+};
+
+const salary = (employees) => {
+  return [...employees].sort((a, b) =>
+    number(b.dataset.salary) - number(a.dataset.salary));
+};
+
+const getEmployees = (employees) => {
+  return [...employees].map(person => (
```

Comment:
There is no need to use the spread operator here because the `map` method doesn't mutate an array but returns a new array.